src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -1057,6 +1057,7 @@ public override void VisitBinaryOperator(IBinaryOperation operation)
 
             LogString($" ({kindStr})");
             LogHasOperatorMethodExpressionCommon(operation.OperatorMethod);
+            var unaryOperatorMethod = ((BaseBinaryOperatorExpression)operation).UnaryOperatorMethod;
```

Comment:
Shouldn't we log this if non-null? Again, can be in a separate PR.

Reply:
> Shouldn't we log this if non-null? Again, can be in a separate PR.

This is not a part of the public API. Changing the output here would cause base-line changes in a lot of tests. There is a PROTOTYPE comment about making this public, then we can deal with this part too. For now, I am comfortable with the coverage for CFG

---
In reply to: [190738488](https://github.com/dotnet/roslyn/pull/27070#discussion_r190738488) [](ancestors = 190738488)